src/NonCumulative/Soundness/Properties/LogRel.agda

```diff
@@ -0,0 +1,84 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Level hiding (_⊔_)
+open import Axiom.Extensionality.Propositional
+
+-- Properties of the gluing models for terms and types
+module NonCumulative.Soundness.Properties.LogRel (fext : ∀ ℓ₁ ℓ₂ → Extensionality ℓ₁ ℓ₂) where
```

Comment:
`ℓ₁ ℓ₂` should be inferrable. make them implicit to avoid filling them in below.